src/std/option-result.md

```diff
@@ -21,5 +21,5 @@ fn main() {
 * `binary_search` returns `Result<usize, usize>`.
   * If found, `Result::Ok` holds the index where the element is found.
   * Otherwise, `Result::Err` contains the index where such an element should be inserted.
-
+* `Option` and `Result` are `enums`, which means we can use enum functions on them. Ex. `first.unwrap_or(0)`.
```

Comment:
What is an "enum function"? `unwrap_or` is just a method defined on `Result` and `Option`.

Reply:
OK,  Let me know if this makes sense.

Reply:
I'm still not sure what you mean by an "enum method". What do these methods have to do with the fact that `Option` and `Result` are enums?